cmd/nerdctl/ps_linux_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+	"gotest.tools/v3/assert"
+)
+
+var (
+	testContainerListName = "test-container-ps"
+)
+
+func prepareTest(t *testing.T) *testutil.Base {
+	base := testutil.NewBase(t)
+
+	base.Cmd("pull", testutil.CommonImage).AssertOK()
+
+	base.Cmd([]string{
+		"run",
+		"-d",
+		"--network", "none",
```

Comment:
Why ?

Reply:
No special meaning, just let it pass on a host without CNI installed.

Reply:
Deleted